python/cudf/cudf/core/udf/__init__.py

```diff
@@ -6,10 +6,16 @@
 from numba.cuda.cudaimpl import lower as cuda_lower
 
 from cudf.core.dtypes import dtype
-from cudf.core.udf import api, row_function, utils
-from cudf.utils.dtypes import STRING_TYPES
-
-from . import groupby_lowering, groupby_typing, masked_lowering, masked_typing
+from cudf.core.udf import api, utils
+
+from . import (
```

Comment:
This is starting to make _lots_ more sense now that more Numba JIT features are being handled in the same way.